utils/MonitoredResource.md

```diff
@@ -0,0 +1,76 @@
+# Monitored Resource
+
+This document is about the monitored resource, what resources are supported and how to automatically
```

Comment:
Shall we mention this document is specifically for exporting to Stackdriver Monitoring?

Reply:
We plan to use this for all the exporters not just SD monitoring.